test/swar/BasicOperations.cpp

```diff
@@ -258,200 +335,173 @@ static_assert(0x0808'0808 == u32(broadcast<8>(SWAR<8, u32>(0x0000'0008))));
 static_assert(0x0B0B'0B0B == u32(broadcast<8>(SWAR<8, u32>(0x0000'000B))));
 static_assert(0x0E0E'0E0E == u32(broadcast<8>(SWAR<8, u32>(0x0000'000E))));
 static_assert(0x6B6B'6B6B == u32(broadcast<8>(SWAR<8, u32>(0x0000'006B))));
-static_assert(0x0808'0808'0808'0808ull == u64(broadcast<8>(SWAR<8, u64>(0x0000'0000'0000'0008ull))));
-
-static_assert(1 == lsbIndex(1<<1));
-static_assert(3 == lsbIndex(1<<3));
-static_assert(5 == lsbIndex(1<<5));
-static_assert(8 == lsbIndex(1<<8));
-static_assert(17 == lsbIndex(1<<17));
-static_assert(30 == lsbIndex(1<<30));
+static_assert(0x0808'0808'0808'0808ull ==
+              u64(broadcast<8>(SWAR<8, u64>(0x0000'0000'0000'0008ull))));
 
+static_assert(1 == lsbIndex(1 << 1));
+static_assert(3 == lsbIndex(1 << 3));
+static_assert(5 == lsbIndex(1 << 5));
+static_assert(8 == lsbIndex(1 << 8));
+static_assert(17 == lsbIndex(1 << 17));
+static_assert(30 == lsbIndex(1 << 30));
 
 /*These tests were not catching errors known to have been present
-static_assert(0x80880008 == greaterEqual<3>(SWAR<4, uint32_t>(0x3245'1027)).value());
-static_assert(0x88888888 == greaterEqual<0>(SWAR<4, uint32_t>(0x0123'4567)).value());
-static_assert(0x88888888 == greaterEqual<0>(SWAR<4, uint32_t>(0x7654'3210)).value());
-static_assert(0x00000008 == greaterEqual<7>(SWAR<4, uint32_t>(0x0123'4567)).value());
-static_assert(0x80000000 == greaterEqual<7>(SWAR<4, uint32_t>(0x7654'3210)).value());
+static_assert(0x80880008 == greaterEqual<3>(SWAR<4,
+uint32_t>(0x3245'1027)).value()); static_assert(0x88888888 ==
+greaterEqual<0>(SWAR<4, uint32_t>(0x0123'4567)).value());
+static_assert(0x88888888 == greaterEqual<0>(SWAR<4,
+uint32_t>(0x7654'3210)).value()); static_assert(0x00000008 ==
+greaterEqual<7>(SWAR<4, uint32_t>(0x0123'4567)).value());
+static_assert(0x80000000 == greaterEqual<7>(SWAR<4,
+uint32_t>(0x7654'3210)).value());
```

Comment:
what the fuck?

Reply:
thanks for pointing out the formatting thing, will try and disable my editor from auto-formatting